.babelrc

```diff
@@ -1,3 +1,8 @@
 {
   "presets": ["es2015"]
+  "env": {
```

Comment:
It looks like there's a missing comma at the end of line 2.  I think that's what's breaking the travis build.  Add the comma and let's see what travis says.